src/index.js

```diff
@@ -1,37 +1,73 @@
-import React, { Component } from "react";
-import ReactDOM from "react-dom";
-import Section from "./Section";
-
-import "./styles.css";
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import Section from './Section';
+import ChocolateQuad from './ChocolateQuad';
+import './styles.css';
+import HappyButton from './HappyButton';
+import Doggie from './Doggie';
+import Food from './Food';
 
 const apiKey = process.env.REACT_APP_API_KEY;
-const searchTerm = "art";
+const searchTerm = 'art';
 const fetchUrl = `https://api.giphy.com/v1/gifs/search?q=${searchTerm}&limit=50&api_key=${apiKey}`;
+const doggieUrl = 'https://dog.ceo/api/breeds/image/random';
+const foodUrl = 'https://foodish-api.herokuapp.com/api/';
 
 class ArtApi extends Component {
-	state = {
-		apiData: [],
-	};
-
-	componentDidMount() {
-		fetch(fetchUrl)
-			.then((res) => res.json())
-			.then((json) => this.setState({ apiData: json.data }));
-	}
-
-	render() {
-		return (
-			<div className="main">
-				<Section apiData={this.state.apiData} />
-				<Section apiData={this.state.apiData} />
-				<Section apiData={this.state.apiData} />
-				<Section apiData={this.state.apiData} />
-			</div>
-		);
-	}
+  constructor(props) {
+    super(props);
+    this.state = {
+      apiData: [],
+      apiData2: '',
+      reloadState: true,
+      api1Call: false,
+      api2Call: false,
+    };
+  }
+
+  fetchApis = () => {
+    const self = this;
+    self.setState({
+      api1Call: false,
+      api2Call: false,
+    });
+
+    fetch(foodUrl)
+      .then((res) => res.json())
+      .then((json) => this.setState({ apiData: json.image }))
+      .then(() => this.setState({ api1Call: true }));
+
+    fetch(doggieUrl)
+      .then((res) => res.json())
+      .then((json) => this.setState({ apiData2: json.message }))
+      .then(() => this.setState({ api2Call: true }));
+  };
+
+  changeMood() {
```

Comment:
The changeMood() method is unnecessary since the logic of this is being handled on line 63 with the conditional rendering.